src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-alert-conditions.mdx

```diff
@@ -0,0 +1,342 @@
+  We recommend using our sliding window aggregation if you're not expecting to have a steady and consistent stream of data but are expecting some dips and spikes in data. 
+
+  You can learn more about sliding window aggregation in [this NRQL tutorial](/docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/create-smoother-charts-with-sliding-windows/) or by watching this video.
+
+  <Video
+    type="youtube"
+    id="-5--8DZynFE"
+  />
+
+  </Collapser>
+  
+  <Collapser
+    id="streaming-method"
+    title="Streaming method"
+  >
+  In general, we recommend using the **event flow** streaming method. This is best for data that comes into your system frequently and steadily. There are specific cases where **event timer** might be a better method to choose, but for your first alert we recommend our default, **event flow**. To better understand which streaming method to choose, we recommend watching this brief video.  
+
+  <Video
+  type="wistia"
+  id="n6nei987ln"
+/>
+
+  </Collapser>
+
+    <Collapser
+    id="delay"
+    title="Delay"
+  >
+
+The delay feature protects you against inconsistent data collection. It gives the alert condition a little wiggle room before deciding to create an incident. If in any given minute your data arrives at New Relic with timestamps from only a single minute, then a low delay setting is enough. On the other hand, if during that minute New Relic receives data points with timestamps from several minutes past or several minutes forward, then your signal is more inconsistent and will need a higher delay setting.
+
+  </Collapser>
+
+  <Collapser
+    id="gap-filling-strategy"
+    title="Gap-filling strategy"
+  >
+  We're creating an alert condition to notify our team of any latency issues with the `WebPortal` application. In this example, our application consistently sends New Relic data. There is a constant stream of signals being sent from our application to New Relic and there is no expected gap in signal so we won't need to select a gap-filling strategy. We generally recommend leaving the **gap-filling strategy** set to **none**. 
+
+  If you have a more inconsistent data set sending signals to New Relic once every twenty-four hours, then we'd recommend customizing this feature based on your team's specific needs. 
+
+  Learn more about gap-filling strategies in our [lost signal docs](/docs/apis/nerdgraph/examples/nerdgraph-api-loss-signal-gap-filling/).
+
+  </Collapser>
+</CollapserGroup>
+
+</Step>
+
+<Step>
+### Set thresholds for alert conditions [#thresholds]
+
+If an alert condition is a container, then thresholds are the rules that each alert condition contains. As data streams into your system, the alert condition searches for any incidents of these rules. If the alert condition sees data coming in from your system that has met all the conditions you've set, then it will create an incident. An incident is a signal that something is off in your system and you should take a look. 
```

Comment:
I'd use a different word than 'contains', since it's defining itself with the word. "...are the rules that each alert condition must follow" or "defines the logic that opens the container"? I'm not sure -- the container metaphor is a little hard for me to understand. when i read 'container' in a docs context, my mind goes to docker containers